community/gnupg2/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+# fake
+export MAKEINFO=true
+touch fig2dev convert ; chmod +x fig2dev convert
+export PATH=$PWD:$PATH
+
+./configure --prefix=/usr \
+            --sbindir=/usr/bin \
+            --libexecdir=/usr/lib/gnupg \
+            --enable-maintainer-mode \
+            --enable-symcryptrun \
+            --enable-gpg-is-gpg2
```

Comment:
Also add `--disable-ldap` and `--disable-nls` (this might also fix the documentation issue).

Reply:
Also `--enable-maintainer-mode` isn't enabled in both Void and Alpine.

Reply:
let me take a wack at this. IIRC i built this one based on arch pkgbuild

Reply:
Arch does things weirdly sometimes. They tend to enable all optional dependencies too. I like to use CRUX, Adelie, Alpine or Void as references when needed (I look at all 4 and cherry-pick what I need). 

You could also look at https://repology.org. Basically, I find it better to look at more than one distributions package if a reference is needed. :+1:

Reply:
got it.